x-pack/filebeat/module/crowdstrike/_meta/fields.yml

```diff
@@ -0,0 +1,42 @@
+- key: crowdstrike
+  title: "Crowdstrike"
+  release: beta
+  description: >
+    Module for collecting Crowdstrike events.
+  fields:
+    - name: crowdstrike
+      type: group
+      description: >
+        Fields for Crowdstrike Falcon event and alert data.
+      fields:
+        - name: metadata
+          title: Metadata fields
+          description: >
+            Meta data fields for each event that include type and timestamp.
+          type: group
+          default_field: false
+          fields:
+            - name: eventType
+              type: text
+              description: >
+                DetectionSummaryEvent or IncidentSummaryEvent
+
+            - name: eventCreationTime
+              type: date
+              description: >
+                The time this event occurred on the endpoint in UTC UNIX_MS format.
+
+            - name: offset
+              type: integer
+              description: >
+                Offset number that tracks the location of the event in stream. This is used to identify unique detection events.
+
+            - name: customerIDString
+              type: text
```

Comment:
In most cases where `text` has been used would probably be better as `keyword`. Then if there were a case where `text` was required we could add a `.text` multi-field. Checkout https://www.elastic.co/guide/en/ecs/current/ecs-conventions.html#_ecs_convention_for_indexing_text_fields.